assignment/scripts/aboutMe.js

```diff
@@ -1,59 +1,106 @@
-
+let friendsPets = 2;
+console.log(friendsPets);
 // 11 - Add two pets to your `pets` variable
-
+pets += 2
+console.log(pets);
 // 12 - Create a constant variable called `allowedPets` and set it to a number value of your choice
-
+const allowedPets = 6;
+console.log(allowedPets);
 // 13 - Create a conditional: if adventurous is true, console log "Adventures are great!", 
 // if it's not true, console log "How about we stay home?"
-
+if (adventurous === true) {
+    console.log("Adventures are great!");
+} else {
+    console.log("How about we stay home?");
+}
 
 // 14 - Create a compound conditional: if luckyNumber is 2 and adventurous is true,
 // console log "Roll the dice!"
-
+if (luckyNumber === 2 && adventurous === true) {
+    console.log("Roll the dice!");
+}
 // 15 - Write a conditional that console logs "I can have more pets!" 
 //if the value of `pets` is less than the value of `allowedPets`,
 // console logs "I have enough pets" if the value of `pets` is equal to the value of `allowedPets`,
 // and console logs "Oh no, I have too many pets!"
 // if the value of `pets` is greater than the value of `allowedPets`.
-
+if (pets < allowedPets) {
+    console.log("I can have more pets!");
+} else if (pets === allowedPets) {
+    console.log("I have enough pets");
+} else if (pets > allowedPets) {
+    console.log("Oh no, I have too many pets!");
+
+}
 // STRETCH GOALS:
 
 // 16 - Make a variable called `mostPets` and a conditional that
 // correctly checks the `pets` and `friendsPets` variables, and
 // assigns the highest value to `mostPets`. There's several possibilities --
 // be sure to think through all the scenarios. 
 // console.log `mostPets` after the conditional has run.
-
+let mostPets;
+if (pets > friendsPets) {
+    mostPets = pets;
+} else if (friendsPets > pets) {
+    mostPets = friendsPets;
+}
+console.log(mostPets);
 // 17 - Write a *switch* statement that logs:
 //      "First is the worst" if your lucky number is 1
 //      "Second is the best" if your lucky number is 2
 //      "Third is the one with the polka dot dress" if your lucky number is 3
 //      Otherwise, log "Luck is what happens when preparation meets opportunity"
 //      You'll need to research how to use switch statements!
-
+switch (luckyNumber) {
+    case 1:
+        console.log("First is the worst");
+        break;
+    case 2:
+        console.log("Second is the best");
+        break;
+    case 3:
+        console.log("Third is the one with the polka dot dress");
+        break;
+    default:
+        console.log("Luck is what happens when preparation meets opportunity");
+        break;
+}
 // 18 -- Rewrite question 13 with a `ternary` operator. You'll need to do research!
+    // 13 - Create a conditional: if adventurous is true, console log "Adventures are great!",
+    // if it's not true, console log "How about we stay home?"  
+adventurous ? console.log("Adventures are great!") : console.log("How about we stay home?");
```

Comment:
The key difference between a ternary  and an if-block and isn't that the ternary is shorter, but that the ternary _is an expression_. That means it outputs a value. So while this code works, a more appropriate use of a ternary would probably save the output to a variable and then log that:

```js
let message = adventurous ? "Adventures are great!" : "How about we stay home?";
console.log(message);
```